app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+use App\Models\Task;
+
+class TaskController extends Controller
+{
+    //
+    function displayTasks() { 
```

Comment:
Good Practice: Always use 'public' explicitly for clarity and adherence to PHP best practices.

`public function displayTasks() {`

do this for all functions